db/migrate/20200115124513_upcase_postcodes_on_application_forms.rb

```diff
@@ -0,0 +1,11 @@
+class UpcasePostcodesOnApplicationForms < ActiveRecord::Migration[6.0]
+  def up
+    ApplicationForm.all.each do |a|
+      a.update!(postcode: a.postcode.upcase) unless a.postcode.nil?
```

Comment:
If we can't use an SQL query we should use `update_column`, which doesn't update the timestamp / add things to the audit log.

Reply:
Ah I've updated to use an SQL query now! Didn't realise the timestamp/audit log thing.